collector/swap_linux.go

```diff
@@ -0,0 +1,125 @@
+// Copyright The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build !noswap
+// +build !noswap
```

Comment:
nit: no need for this one, we already require Go `>=1.24`

Reply:
Thanks, removed.

Reply:
Sorry, I meant no need to have `// +build !noswap` (old syntax) but we can keep  `//go:build !noswap`